src/index.js

```diff
@@ -178,11 +178,11 @@ addNewListForm.addEventListener('submit', (e) => {
 });
 
 // Add event listener to the "Clear all completed" button
-// clearTaskBtn.addEventListener('click', () => {
-//   tasks = tasks.filter((task) => !task.completed); // Remove completed tasks from the tasks array
-//   updateIndexes(); // Update the indexes of remaining tasks
-//   saveAndRender();
-// });
+clearTaskBtn.addEventListener('click', () => {
+  tasks = tasks.filter((task) => !task.completed); // Remove completed tasks from the tasks array
+  updateIndexes(); // Update the indexes of remaining tasks
+  saveAndRender();
+});
```

Comment:
As part of the requirement for this project, you should have a new JavaScript file imported as a module:

* it should contain methods related to the status updates (`completed`: `true` / `false`).